opennlp-tools/src/main/java/opennlp/tools/language/LanguageCode.java

```diff
@@ -0,0 +1,85 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package opennlp.tools.language;
+
+import java.util.Objects;
+
+/**
+ * Represents a language.
+ *
+ */
+public final class LanguageCode {
+
+  public static final LanguageCode ARABIC = new LanguageCode("aao");
+  public static final LanguageCode BENGALI = new LanguageCode("ben");
+  public static final LanguageCode DUTCH = new LanguageCode("nld");
+  public static final LanguageCode ENGLISH = new LanguageCode("eng");
+  public static final LanguageCode FRENCH = new LanguageCode("fra");
+  public static final LanguageCode GERMAN = new LanguageCode("deu");
+  public static final LanguageCode HAUSA = new LanguageCode("hau");
+  public static final LanguageCode ITALIAN = new LanguageCode("ita");
+  public static final LanguageCode JAPANESE = new LanguageCode("jpn");
+  public static final LanguageCode KOREAN = new LanguageCode("kor");
+  public static final LanguageCode MALAY = new LanguageCode("mal");
+  public static final LanguageCode PERSIAN = new LanguageCode("prp");
+  public static final LanguageCode PORTUGUESE = new LanguageCode("por");
+  public static final LanguageCode PUNJABI = new LanguageCode("pan");
+  public static final LanguageCode RUSSIAN = new LanguageCode("rus");
+  public static final LanguageCode SPANISH = new LanguageCode("spa");
+  public static final LanguageCode SWAHILI = new LanguageCode("swa");
+
+  private String languageCode;
+
+  /**
+   * Creates a new <code>LanguageCode</code>.
+   * @param languageCode The language code.
+   */
+  public LanguageCode(String languageCode) {
+    this.languageCode = languageCode;
+  }
+
+  /**
+   * Gets the language code.
+   * @param languageCode The language code.
+   */
+  public String getLanguageCode() {
+    return languageCode;
+  }
+
+  @Override
+  public String toString() {
+    return languageCode;
+  }
+
+  @Override
+  public boolean equals(Object obj) {
+    if (obj instanceof LanguageCode) {
```

Comment:
Would be good if we can follow the "standard pattern" we have in all other places in OpenNLP for equals impl.